pkgs/applications/misc/rofi/default.nix

```diff
@@ -23,6 +23,9 @@
   check,
   glib,
   buildPackages,
+
+  imeSupport ? false,
```

Comment:
dunno its impact on closure size but if it's not too much I would say we could always use ime. 
It's 2025 and IME + wayland should be defaults IMO.

Reply:
Thanks for the review, yeah I wondered about having it by default, I pushed a new version without the `imeSupport` flag